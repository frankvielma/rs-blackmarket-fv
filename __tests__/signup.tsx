import { render } from '@testing-library/react'
import Signup from '@/app/signup/page'

it('renders signup unchanged', () => {
  const { container } = render(<Signup />)
  expect(container).toMatchSnapshot()
})
