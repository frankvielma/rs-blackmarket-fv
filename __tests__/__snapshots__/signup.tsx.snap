// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signup renders signup unchanged 1`] = `
<div>
  <main
    class="flex h-[100vh] flex-col bg-[url('/images/bm_wallpaper.jpg')] bg-cover bg-center bg-no-repeat px-6 py-16 text-base text-dark-violet md:px-28"
  >
    <div
      class="h-[496px] max-w-[328px] rounded-lg border-black bg-white px-[34px] md:h-[572px] md:max-w-[360px]"
    >
      <h1
        class="sr-only"
      >
        Sign up
      </h1>
      <div
        class="pb-5 pt-[40px] md:p-[47px] md:pb-7"
      >
        <img
          alt="Black Market Logo"
          class="h-auto w-full"
          data-nimg="1"
          decoding="async"
          height="0"
          loading="lazy"
          sizes="100vw"
          src="/_next/image?url=%2Fimages%2Fbm_logo.png&w=3840&q=75"
          srcset="/_next/image?url=%2Fimages%2Fbm_logo.png&w=640&q=75 640w, /_next/image?url=%2Fimages%2Fbm_logo.png&w=750&q=75 750w, /_next/image?url=%2Fimages%2Fbm_logo.png&w=828&q=75 828w, /_next/image?url=%2Fimages%2Fbm_logo.png&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2Fbm_logo.png&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2Fbm_logo.png&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2Fbm_logo.png&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2Fbm_logo.png&w=3840&q=75 3840w"
          style="color: transparent;"
          width="0"
        />
      </div>
      <form>
        <div
          class="my-4 flex flex-col"
        >
          <label
            class="text-sm md:text-base"
            for="email"
          >
            Email
          </label>
          <input
            class="border custom-focus h-[34px] w-auto rounded-md border border-dark-violet ps-3 placeholder:text-base placeholder:text-dark-gray hover:border-hover active:outline-none active:ring-2 active:ring-offset-2 md:h-[44px]"
            id="email"
            pattern="[a-z0-9._%+-]+@[a-z0-9.-]+.[a-z]{2,}$"
            placeholder="Type your email"
            required=""
            type="email"
          />
          <span />
        </div>
        <div
          class="relative my-4"
        >
          <label
            class="text-sm md:text-base"
            for="password1"
          >
            Password
          </label>
          <div
            class="flex flex-col"
          >
            <input
              class="border custom-focus h-[34px] w-auto rounded-md border border-dark-violet ps-3 placeholder:text-base placeholder:text-dark-gray hover:border-hover active:outline-none active:ring-2 active:ring-offset-2 md:h-[44px]"
              id="password1"
              pattern=".{8,}"
              placeholder="Type your password"
              required=""
              type="password"
            />
            <span />
            <button
              class="custom-focus absolute right-4 mt-2 md:mt-4"
              type="button"
            >
              <span
                class="sr-only"
              >
                Show password
              </span>
              <img
                alt="Visibility"
                class="h-auto w-4"
                data-nimg="1"
                decoding="async"
                height="0"
                loading="lazy"
                src="/icons/visibility_off.svg"
                style="color: transparent;"
                width="0"
              />
            </button>
          </div>
        </div>
        <div
          class="relative my-4"
        >
          <label
            class="text-sm md:text-base"
            for="password2"
          >
            Confirm password
          </label>
          <div
            class="flex flex-col"
          >
            <input
              class="border custom-focus h-[34px] w-auto rounded-md border border-dark-violet ps-3 placeholder:text-base placeholder:text-dark-gray hover:border-hover active:outline-none active:ring-2 active:ring-offset-2 md:h-[44px]"
              id="password2"
              pattern=".{8,}"
              placeholder="Confirm your password"
              required=""
              type="password"
            />
            <span />
            <button
              class="custom-focus absolute right-4 mt-2 md:mt-4"
              type="button"
            >
              <span
                class="sr-only"
              >
                Show password
              </span>
              <img
                alt="Visibility"
                class="h-auto w-4"
                data-nimg="1"
                decoding="async"
                height="0"
                loading="lazy"
                src="/icons/visibility_off.svg"
                style="color: transparent;"
                width="0"
              />
            </button>
          </div>
        </div>
        <div
          class="flex flex-col"
        >
          <button
            class="h-[44px] rounded-md border text-center text-base
    font-bold
    bg-black text-white border-white
    bg-light-gray text-dark-gray outline-none cursor-not-allowed
  "
            type="button"
          >
            Sign up
          </button>
        </div>
        <span
          class="text-red-700"
        />
      </form>
      <div
        class="mt-5 text-center"
      >
        By signing up, you accept the
         
        <a
          class="link-focus link-hover link-active"
          href="/data_policy"
        >
          Data Policy
        </a>
         
        and the
         
        <a
          class="link-focus link-hover link-active"
          href="/cookies_policy"
        >
          Cookies Policy.
        </a>
      </div>
      <div
        class="mt-5 text-center"
      >
        Already have an account?
         
        <a
          class="link-focus link-hover link-active"
          href="/login"
        >
          Log in
        </a>
      </div>
    </div>
  </main>
</div>
`;
